js/components/discussions/thread.vue

```diff
@@ -97,10 +100,10 @@ export default {
             return `discussion-${this.discussion.id}`;
         },
         createdDate() {
-            return moment(this.discussion.created).format('LL')
+            return moment(this.discussion.created).format('LL');
```

Comment:
Was it intended to modify this line to add a semicolon while the function `formatDate` is added below with a very similar code:

```
return moment(val).format('LL');
```

Is it to avoid an indirection that `formatDate` isn't used here ?

Reply:
(it's minor, I marked it as "change requested" to make it visible because I'm not sure of the etiquette of this repository yet)

Reply:
yep it was intended but only to comply with @abulte comment about not using StandardJS (which promote not using semi).

The best workaround for this case would be to implement a global filter but I am still a bit shy about code impact.